### ISSUE TITLE
`Required` and `NotRequired` cannot be used without a type argument in runtime code

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#2723]
**Describe the bug**
Given the following code:

```python
from __future__ import annotations

from typing import TypedDict, get_args, get_origin, get_type_hints
from typing_extensions import NotRequired, Required


class SomeDict(TypedDict):
    id: Required[int]
    permissions: NotRequired[int]


hints = get_type_hints(SomeDict)

for name, annotation in hints.items():
    origin = get_origin(annotation)

    if origin is Required:
        args = get_args(annotation)

        print(f'{name} required:', args)
    elif origin is NotRequired:
        args = get_args(annotation)

        print(f'{name} not required:', args)
```

Errors are produced on lines 17 (`Expected a single type argument after "Required"`) and 21 (`Expected a single type argument after "Required"`).

**Expected behavior**
No errors are produced.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command line

Which version?
1.1.196

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.197, which I just published. It will also be included in the next release of pylance.
